AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/CodeGen/States/MigrateSignInState.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+enum MigrateSignInState: State {
+
+    case notStarted
+    case initiating(SignInEventData)
```

Comment:
can we use `signingIn` instead of `initiating`

Reply:
Changed